src/main/java/org/apache/commons/csv/CSVParser.java

```diff
@@ -438,6 +440,44 @@ public CSVParser(final Reader reader, final CSVFormat format, final long charact
         this.recordNumber = recordNumber - 1;
     }
 
+    /**
+     * Return the parsed CSV content of currently parsed line up until this method is called.
```

Comment:
If it's just a string I'm not sure it's "parsed". Maybe just "Return the text of the current line up until this method is called."

Reply:
What we have in the buffer up until the error point is parsed as tokens. So I feel this is kind of OK.